File: solutions/cold.py
# Author: Thomas Vu
# Email: thomas.vu@ucalgary.ca
# Feel free to send any questions about this problem to the email above
# or ask in the CPC discord. (discord.gg/MEXwfze)

n = int(input())
temperatures = [int(i) for i in input().split()]
tempsBelowZero = 0
for temp in temperatures:
    if temp < 0:
        tempsBelowZero += 1
print(tempsBelowZero)


# Short version:

# input()
# print(input().count("-"))

File: solutions/Cold.java
/*
 Author: Martin Tran
 Email: martin.tran@ucalgary.ca

 Feel free to send any questions about this problem to the email
 above or ask in the CPC discord. (discord.gg/MEXwfze)
 */
import java.util.Scanner;

public class Cold {

    public static void main(String[] args) {
	Scanner sc = new Scanner(System.in);
	
	int count = 0;
	int n = sc.nextInt(); 
	
	for(int i = 0; i < n; i++){
	    int next = sc.nextInt();
	    
	    if(next < 0)
		count ++;
	}
	
	System.out.println(count);
	sc.close();	
    }   
}
